script.js

```diff
@@ -47,4 +50,119 @@ buttonElement.addEventListener('click', function (event) {
     subElement.classList.toggle('active');
 });
 
+buttonRight.addEventListener('click', function (){
+    move('right');
+}); 
+buttonLeft.addEventListener('click', function (){
+    move('left');
+});
+
+let images = document.querySelectorAll('.carousel object');
+let slider = [];
+for (let i = 0; i < images.length; i++) {
+    slider[i] = images[i].data;
+    images[i].remove();
+}
+let boxes = document.querySelectorAll('.carousel__box');
+for (let i = 0; i < boxes.length; i++) {
+    boxes[i].remove();
+}
+
+let carouselTextObject = document.createElement('div');
+carouselTextObject = carouselText;
+
+let step = 0;
+let carousel =  Array.from(document.querySelectorAll('.carousel__box'));
```

Comment:
A big no to getting elements using class. Use ID and then `.children`

Reply:
Ok, changed it: let carousel =  Array.from(carouselBox.children);

Reply:
No, you are still using class to get elements remove the querySelectorAll and replace with getElementById

Reply:
> No, you are still using class to get elements remove the querySelectorAll and replace with getElementById

Ok, fixed!

Reply:
It's not fixed.

Reply:
Already done.